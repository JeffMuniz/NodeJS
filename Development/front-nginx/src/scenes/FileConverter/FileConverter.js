import React from "react";
import { Col } from "react-styled-flexboxgrid";
import { ContainerWrapper } from "@base";

import FormContainer from "./Form/Form.container";
import { RowStyled, Div } from "./FileConverter.styles";

const FileConverter = () => (
  <ContainerWrapper title="Arquivo TXT">
    <Div>
      Envie o arquivo TXT para que seja automaticamente convertido para o
      formato mac.
    </Div>
    <RowStyled>
      <Col xs={12}>
        <FormContainer />
      </Col>
    </RowStyled>
  </ContainerWrapper>
);

export default FileConverter;
