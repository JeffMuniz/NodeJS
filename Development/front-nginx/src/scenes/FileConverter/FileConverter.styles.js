import styled from "styled-components";
import { Row } from "react-styled-flexboxgrid";

export const RowStyled = styled(Row)`
  margin-top: 30px;
`;

export const Div = styled.div``;
