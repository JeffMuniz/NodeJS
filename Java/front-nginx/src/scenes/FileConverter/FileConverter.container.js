import React from "react";

import View from "./FileConverter";

const FileConverterContainer = () => <View />;

export default FileConverterContainer;
