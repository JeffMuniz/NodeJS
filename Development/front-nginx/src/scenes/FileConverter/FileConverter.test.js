import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import FileConverter from "./FileConverter";

jest.mock("@base", () => ({
  ContainerWrapper: "ContainerWrapper",
}));
jest.mock("@common", () => ({
  SvgIcon: "SvgIcon",
  Button: "Button",
}));

const defaultProps = {
  navigator: {},
  history: {},
  goToDocumentModel: jest.fn(),
};

describe("Order", () => {
  it("Should render empty dashboard", async () => {
    // given
    const renderer = new ShallowRenderer();

    // when
    await renderer.render(<FileConverter {...defaultProps} />);
    const result = renderer.getRenderOutput();

    // then
    expect(result).toMatchSnapshot();
  });
});
