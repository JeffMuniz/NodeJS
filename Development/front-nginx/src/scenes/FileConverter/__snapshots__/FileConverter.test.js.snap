// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order Should render empty dashboard 1`] = `
<ContainerWrapper
  title="Arquivo TXT"
>
  <styled.div>
    Envie o arquivo TXT para que seja automaticamente convertido para o formato mac.
  </styled.div>
  <Styled(Row)>
    <Col
      xs={12}
    >
      <Connect(FormContainer) />
    </Col>
  </Styled(Row)>
</ContainerWrapper>
`;
